main.js

```diff
@@ -18,28 +23,61 @@ const name2 = {
     }
 };
 
-function createPlayer(className, player) {
-    const playerElement = document.createElement('div');
-    playerElement.classList.add(className);
-    const progressbarEl = document.createElement('div');
-    progressbarEl.classList.add('progressbar');
-    const lifeElement = document.createElement('div');
-    lifeElement.classList.add('life');
+function createElement(tag, className) {
+    const tagElement =  document.createElement(tag);
+    if (className) {
+        tagElement.classList.add(className);
+    }
+    return tagElement;
+}
+
+function createPlayer(player) {
+    const playerElement = createElement('div', 'player'+player.player);
+    const progressbarEl = createElement('div', 'progressbar');
+    const lifeElement = createElement('div', 'life');
     lifeElement.style.width = `${player.hp}%`;
-    const nameElement = document.createElement('div');
+    const nameElement = createElement('div', 'name');
     nameElement.innerText = player.name;
-    nameElement.classList.add('name');
     progressbarEl.appendChild(lifeElement);
     progressbarEl.appendChild(nameElement)
     const characterElement = document.createElement('div');
     characterElement.classList.add('character');
-    const imageElement = document.createElement('img');
+    const imageElement= createElement('img');
     imageElement.src = player.img;
     characterElement.appendChild(imageElement);
     playerElement.appendChild(progressbarEl);
     playerElement.appendChild(characterElement);
+    return playerElement;
+}
+
+arenasElement.appendChild(createPlayer(player1));
+arenasElement.appendChild(createPlayer(player2));
 
-    document.querySelector('.arenas').appendChild(playerElement);
+function playerLoose(name) {
+    const  loseTitle = createElement('div', 'loseTitle');
+    loseTitle.innerText = name + ' lose';
```

Comment:
тоже используй бектики раз уже знаком с ними
```suggestion
    loseTitle.innerText = `${name} lose`;
```